### ISSUE TITLE
Support for environment.yml

### ISSUE THREAD

**Issue by amfarrell** [kennethreitz/conda-buildpack#15]
Conda now supports an environment specification file that can include both pip and conda packages and thereby replace the two files requirements.txt and conda-requirements.txt. An example of such a file can be found at https://raw.githubusercontent.com/memex-explorer/memex-explorer/master/environment.yml

This would also solve issue #11, as environment.yml files can specify channels.

@kennethreitz, would you accept a PR to support this, or do you think it would be better to do this as a separate conda-env-buildpack?

**Comment by amfarrell** [comment]
One question this raises is: What do we do if an environment.yml is present alongside a requirements.txt
There are three options for handling this:
1) The environment.yml takes precedence and the requirements.txt is ignored.
2) An error is raised if both are present
3) Packages from environment.yml are installed first and then from requirements.txt are installed.

**Comment by birdsarah** [comment]
Agreed @amfarrell, this would be great, I'm much preferring using `environment.yml` to specify environments than `conda-requirements.txt`

**Comment by talumbau** [comment]
+1.
